pkgs/applications/office/docear/default.nix

```diff
@@ -0,0 +1,48 @@
+{stdenv, fetchurl, runtimeShell
+, oraclejre
+, antialiasFont ? true
+}:
+
+stdenv.mkDerivation rec {
+  pname = "docear";
+  version = "1.2";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "http://docear.org/downloads/docear_linux.tar.gz";
```

Comment:
This url isn't stable so once upstream version bumps this will break.

Reply:
Yes, but unfortunately, as mentioned in the commit message and the website, stable urls are only available for older versions.

Reply:
@unode I would suggest filing an issue upstream. Hopefully they will be receptive so we can avoid breakage.

Reply:
This is unlikely given that the sourceforge page includes all older versions and also: https://sourceforge.net/projects/docear/files/1.x/%21%21%21___download_the_latest_releases_only_from_www.docear.org___%21%21%21.txt/download